client/my-sites/earn/home.tsx

```diff
@@ -223,7 +224,7 @@ const Home: FunctionComponent< ConnectedProps > = ( {
 
 		const learnMoreLink = isFreePlan
```

Comment:
Is there something different here about the whitespace? The start of this line should be aligned with the start of `const body = hasConnectedAccount ? hasConnectionBody : noConnectionBody;` I think